scripts/west_commands/runners/openocd.py

```diff
@@ -118,7 +122,8 @@ def do_run(self, command, **kwargs):
 
         self.cfg_cmd = []
         if self.openocd_config is not None:
-            self.cfg_cmd = ['-f', self.openocd_config]
+            self.cfg_cmd = ['-f', self.openocd_config,
+                            '-f', 'openocd.zephyr.cfg']
```

Comment:
Can we make this work like the jlink runner, where it's only passed to openocd if CONFIG_DEBUG_THREAD_INFO=y? You can search in the jlink.py source code for `self.thread_info_enabled` to see how it's done.